### PR TITLE
Feature/sonar cloud cleanup

### PR DESCRIPTION
**Pull request by lmalenfant**
Hi Carole,

Our code smells are down to 397:
https://sonarcloud.io/summary/overall?id=VirtualPhotonics_VTS

Hopefully I didn't break anything :)

### CODE REVIEW

**Comment by hayakawa16** [comment]
Hi Lisa,

I will review code changes, then run branch code on linux before
I accept.

Carole

File: src/Vts/Modeling/ForwardSolvers/TwoLayerSDAForwardSolver.cs
Comment (hayakawa16) [inline src/Vts/Modeling/ForwardSolvers/TwoLayerSDAForwardSolver.cs:1]: FFT is an acronym for fast Fourier Transform.  So wouldn't camel case be "fFt"?  Because I think we changed "pMC"
to be "pMc".  Not sure.  It is your call.

File: src/Vts/MonteCarlo/Helpers/SourceDefaults.cs
Comment (hayakawa16) [inline src/Vts/MonteCarlo/Helpers/SourceDefaults.cs:1]: Nice!

File: src/Vts/Modeling/ForwardSolvers/TwoLayerSDAForwardSolver.cs
Comment (lmalenfant) [reply src/Vts/Modeling/ForwardSolvers/TwoLayerSDAForwardSolver.cs:1]: Yes I think you're right, I'll definitely change that.

**Comment by lmalenfant** [comment]
Hi Carole,

I just realized that I sent you the link to the Sonar Cloud analysis but since I added it to the repo, you can now see the analysis directly in this PR. It is in the "Checks" tab at the top. We still have quite a lot of duplications so I had to increase the percentage to 20% to pass the quality gate until we can address it.

**Review by hayakawa16 — APPROVED** [review]
Going over your changes in person was *very* helpful to me because I could review these file changes quickly and know what they were about.  So thanks for doing that.
I updated to branch on linux, ran infile_one_layer_all_detectors.txt, then post-processed using load_results_script.m.  All looks great.  Nice work!
I think it might be good to merge together when I merge the updated master into all branches.

**Comment by hayakawa16** [comment]
Hi Lisa,

I just viewed it.  Is that a new capability to GitHub?  Or is it just
because you added it to the repo.  That is nice!  I *do* still like
the screen capture I've been keeping because will this tab and 
information
go away when this PR is closed?

Carole

**Comment by lmalenfant** [comment]
The Sonar Cloud scanning has been a part of GitHub for a while, it has always been connected but I have to run the analysis separately. Eventually, I hope to have it working with a pipeline but I have to automate the code coverage results first (I'm almost there). You're right, this will go away when we merge the PR and the history is available but we don't view it the same way on Sonar Cloud so keeping a screenshot of the progress is still good.

I'll fix that one name and then merge, yes, let's merge master into the branches together.

**Comment by lmalenfant** [comment]
Hi Carole (@hayakawa16),

I made the name change, could you verify that it looks right? I believe it should be fFtTimeSequence.

**Comment by hayakawa16** [comment]
Hi Lisa,

I think that is correct, however when I wrote it out it seemed funny
to me and so I looked in the code for other acronyms and what we did.
That is when I found pMc for pMC.  I deferred to you because you know
more about the right thing to do here.

Carole

**Pull request merged**
